### ISSUE TITLE
current version numbering is incompatible with setuptools

### ISSUE THREAD

**Issue by emtiu** [getmail6/getmail6#21]
When using setuptools to install getmail6, the script spouts this warning:
```
/usr/local/lib/python3.7/site-packages/setuptools/dist.py:476: UserWarning: Normalizing '6.03' to '6.3'`
```

This affects the location of metadata handled by setuptools, like these files:
```
/usr/local/lib/python3.7/site-packages/getmail-6.3-py3.7.egg-info/PKG-INFO
/usr/local/lib/python3.7/site-packages/getmail-6.3-py3.7.egg-info/SOURCES.txt
/usr/local/lib/python3.7/site-packages/getmail-6.3-py3.7.egg-info/dependency_links.txt
/usr/local/lib/python3.7/site-packages/getmail-6.3-py3.7.egg-info/top_level.txt
```

Judging from the answer to [this StackOverflow question](https://stackoverflow.com/questions/60323764/python-setup-tools-normalizes-version-name-based-on-common-naming-convention), we're probably in the wrong here, because `6.03` is never acceptable to Python's setuptools.

**I strongly suggest** that we move the versioning scheme to 6.0.4 at the next opportunity, because this could cause us all sorts of trouble down the line (like when an actual version 6.3 normalizes to the same version string as 6.03).

**Also note that** for Python internally, the package is still called `getmail` and not `getmail6`. I'm not sure if that's a problem. The last major revision that Charles Cazabon published was 5.0 in 2017. (If this is a concern, we might move it to a separate issue.)

**Comment by Remmy** [comment]
Michael, would it not be better to make the next version number 6.4 then, and count upwards from there? Seeing how 6.03 has already been normalized to 6.3, that seems the safer option to me.

**Comment by Remmy** [comment]
> **Also note that** for Python internally, the package is still called `getmail` and not `getmail6`. I'm not sure if that's a problem. The last major revision that Charles Cazabon published was 5.0 in 2017. (If this is a concern, we might move it to a separate issue.)

Can you expand on this? What will break if we do/don't change this?

**Comment by rpuntaie** [comment]
I introduced the 0, to make e.g. `6.03<6.14` using lexicographic order, which would not be the case for `6.3`, and missed the warning. Version numbers are ordered by number value only (a pity, since both orders could be made to match).
[Lexicographic sort order only applies to versions with the same numerical compounds but different appended strings]
(https://github.com/semver/semver.org/issues/39).
I agree with @Remmy  to continue with 6.4, else we would change the version ordering, which is worse.
Also, 6.xy is a continuation of Cazabon's style. We better just avoid the leading 0.

Regarding python name `getmail`. I think, this is no problem. The scripts do have the same name, too.
It was meant just as port, as drop-in replacement.
